bubuku/features/rebalance/change.py

```diff
@@ -155,6 +156,28 @@ def run(self, current_actions) -> bool:
             return not self._balance()
         return True
 
+    @staticmethod
+    def get_rebalance_data(zk: BukuExhibitor, broker_ids):
+        current_actions = []
+        with zk.lock('hand_rebalance'):
+            current_actions = zk.get_running_changes()
+        change = OptimizedRebalanceChange(zk, broker_ids)
+        while change.run(current_actions) and change.state != OptimizedRebalanceChange._BALANCE:
+            pass
+        disk_stats = zk.get_disk_stats()
+
+        def _get_size(v):
+            for ds in disk_stats.values():
+                size = ds['topics'].get(v[0], {}).get(str(v[1]))
+                if size is not None and size > 0:
+                    return size
+            return 0
+
+        actions_sorted_by_size = sorted(change.action_queue, key=_get_size)
+        # actions_sorted_by_size = change.action_queue
```

Comment:
Well, there is actually problem here. It will sort only changes, not the source data. IT works when it's a handmade process, but with real process one should put more effort to do it better, or at least add comment about that in code